src/operator/nn/mkldnn/mkldnn_base-inl.h

```diff
@@ -312,7 +312,8 @@ inline static mkldnn::memory::desc GetFCWeightDesc(const NDArray &arr, int dtype
   for (size_t i = 0; i < dims.size(); i++) dims[i] = arr.shape()[i];
   auto format = mkldnn::memory::format_tag::any;
   // for batch 256 alexnet benchmark test
-  if (dims.size() == 2) {
+  const bool brgemm_disabled = dmlc::GetEnv("MXNET_MKLDNN_DISABLE_BRGEMM_FC", true);
+  if (dims.size() == 2 && brgemm_disabled) {
```

Comment:
Could you please provide more benchmarking data with different m/n/k and formats?

BTW, actually `brgemm_disabled` looks misleading to me. According to the code change, i would rather call the flag `force_plain_format` or `force_ab_format`.